pkg/cmd/repo/edit/edit.go

```diff
@@ -379,23 +395,26 @@ func interactiveRepoEdit(opts *EditOptions, r *api.Repository) error {
 			}
 			opts.Edits.EnableProjects = &a
 		case optionVisibility:
+			cs := opts.IO.ColorScheme()
+			fmt.Fprintf(opts.IO.ErrOut, "%s Danger zone: changing repository visibility can have unexpected consequences; consult https://gh.io/setting-repository-visibility before continuing.\n", cs.WarningIcon())
```

Comment:
Just wondering why you chose `ErrOut` here. When we have TTYs I always feel a bit like everything should go to `Out` unless it's an actual error, but it seems very gray. I'm never really sure though.

In any case it's not a blocker and we can always change it later since this is interactive.

Reply:
> Just wondering why you chose `ErrOut` here. When we have TTYs I always feel a bit like everything should go to `Out` unless it's an actual error, but it seems very gray. I'm never really sure though.

I totally feel how big of a gray area this is! 💯 Let me capture my thoughts but here is [`cli/cli` code use of `ErrOut`](https://github.com/search?q=repo%3Acli%2Fcli%20ErrOut&type=code).

Suffice to say, I see informational output like this similar to [CLIG](https://clig.dev/#the-basics) guidance as something that really isn't machine readable content but messaging / informational:

> **Send output to stdout**. The primary output for your command should go to `stdout`. Anything that is machine readable should also go to stdout—this is where piping sends things by default.
> 
> **Send messaging to stderr**. Log messages, errors, and so on should all be sent to `stderr`. This means that when commands are piped together, these messages are displayed to the user and not fed into the next command.

If someone was redirecting the output of this command somewhere, I wouldn't want this info to be swallowed up.

Reply:
> If someone was redirecting the output of this command somewhere, I wouldn't want this info to be swallowed up or treated differently if in non-TTY usage.

Right I get all that generally, but this **is** only TTY interactive usage. You can't get to these new messages otherwise, right?